src/proto/single/ent.rs

```diff
@@ -74,6 +91,135 @@ impl JobBuilder {
     }
 }
 
+/// Info on job execution progress (retrieved).
+///
+/// The tracker is guaranteed to get the following details: the job's id (though
+/// they should know it beforehand in order to be ably to track the job), its last
+/// know state (e.g."enqueued", "working", "success", "unknown") and the date and time
+/// the job was last updated. Additionally, information on what's going on with the job
+/// ([desc](struct.ProgressUpdate.html#structfield.desc)) and completion percentage
+/// ([percent](struct.ProgressUpdate.html#structfield.percent)) may be available,
+/// if the worker provided those details.
+#[derive(Debug, Clone, Deserialize)]
+pub struct Progress {
+    /// Id of the tracked job.
+    pub jid: String,
+
+    /// Job's state.
+    pub state: String,
+
+    /// When this job was last updated.
+    #[serde(deserialize_with = "parse_datetime")]
+    pub updated_at: Option<DateTime<Utc>>,
+
+    /// Percentage of the job's completion.
+    pub percent: Option<u8>,
```

Comment:
Is this actually an integer and not a floating point number?

Reply:
They use ints across the doc [examples](https://github.com/contribsys/faktory/wiki/Ent-Tracking#setup).

The go client has got [`int`](https://github.com/contribsys/faktory/blob/main/client/tracking.go#L41) for this field.